django/forms/fields.py

```diff
@@ -601,6 +602,7 @@ def has_changed(self, initial, data):
 
 
 class ImageField(FileField):
+    default_validators = [validate_image_file]
```

Comment:
It's not entirely clear to me how this will interact with the validation in `to_python()`. Have you looked at this?